poh/src/poh_recorder.rs

```diff
@@ -561,30 +563,37 @@ impl PohRecorder {
                 .leader_first_tick_height_including_grace_ticks
                 .is_none()
             {
-                self.tick_overhead_us += timing::duration_as_us(&now.elapsed());
                 return;
             }
 
-            let entry = Entry {
-                num_hashes: poh_entry.num_hashes,
-                hash: poh_entry.hash,
-                transactions: vec![],
-            };
-            self.tick_overhead_us += timing::duration_as_us(&now.elapsed());
-
-            let now = Instant::now();
-            self.tick_cache.push((entry, self.tick_height));
-            let _ = self.flush_cache(true);
-            self.flush_cache_tick_us += timing::duration_as_us(&now.elapsed());
-            let target_time = target_time.unwrap();
-            // sleep is not accurate enough to get a predictable time.
-            // Kernel can not schedule the thread for a while.
-            let started_waiting = Instant::now();
-            while Instant::now() < target_time {
-                // TODO: a caller could possibly desire to reset or record while we're spinning here
-                std::hint::spin_loop();
-            }
-            self.total_sleep_us += started_waiting.elapsed().as_nanos() as u64 / 1000;
+            self.tick_cache.push((
+                Entry {
+                    num_hashes: poh_entry.num_hashes,
+                    hash: poh_entry.hash,
+                    transactions: vec![],
+                },
+                self.tick_height,
+            ));
+
+            let (_flush_res, flush_cache_and_tick_time) =
+                Measure::this(|_| self.flush_cache(true), (), "FlushCacheAndTick");
```

Comment:
in most places we use snake-case for the measure tag names, can we match that convention?

Reply:
This is just for naming the metric, we don't really follow any convention there in the codebase. It's not reported to metrics at all, is it really worth updating?

Reply:
I switched mine here to be consistent: https://github.com/solana-labs/solana/pull/22744, can just update these there too

Reply:
Updated here: https://github.com/solana-labs/solana/pull/22744/commits/626fa74809f2fa84939d95afa2b29cfb92af950d